### DISCUSSION TITLE
能否重写.gitignore，使其变的更加的整洁？

### DISCUSSION THREAD

**Discussion by RobinEllison** [TheNetAdmin/zjuthesis#262] [General]
现在的.gitignore感觉写的过于的繁琐，此外一些非文字的文件，比如存在figure/下面的所有图片，我建议全部忽略。

**Comment by TheNetAdmin** [comment]
我没太搞懂你说的整洁是怎么个整洁，能举例说明吗？

Ignore图片不是什么好practice ，万一你本地文件崩了，你从git上clone个新的论文，里面没有图片，那岂不是更麻烦

**Comment by RobinEllison** [comment]
个人感觉 .gitignore 里面写的规则比较多，而需要有版本控制的文件, 一方面包括body, config, docs, page, script文件夹下的文件，另一方面包括主目录下的zjuthesis.tex, zjuthesis.tex, LICENSE, README.md, .gitignore 文件。我们可以直接track这些文件或者文件夹即可，忽略其它。我觉得figure下的文件主要是图片，不适合用git做版本控制，比如我只是调整了一张图片，我就要加一个commit, 比较的繁琐。另一方面是很少有人会把自己的论文挂到github上去，也就没有必要git clone的必要，如果能够自己搭建git服务器（比如gitlab）,我想有能力做这件事的人，肯定有能力自己去修改.gitignore，来预防你说的本地文件奔溃的情况。
